recipes/eccodes/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
I'm not sure. In the sources there is a "windows"-folder with .vcproj files for several grib/bufr programs. Shall we try?

What has to go into the bld.bat?